src/main/kotlin/no/nav/syfo/kafka/KafkaTask.kt

```diff
@@ -15,8 +14,6 @@ inline fun <reified ConsumerRecordValue> launchKafkaTask(
     launchBackgroundTask(
         applicationState = applicationState
     ) {
-        LOG.info("Setting up kafka consumer for ${ConsumerRecordValue::class.java.simpleName}")
-
```

Comment:
Vil vi ha en erstatter for denne?

Reply:
Tror vi klarer oss godt uten den loggingen.